ebos/equil/initstateequil.hh

```diff
@@ -1819,14 +1819,14 @@ private:
         auto psat   = PhaseSat { materialLawManager, this->swatInit_ };
         auto vspan  = std::array<double, 2>{};
 
+        int regionIsEmpty[rec.size()] = {0};
```

Comment:
Tiny nit: This should probably be a `std::vector<int>` instead.  Variable-length arrays are a GCC extension.

Reply:
The ` comm.sum(regionIsEmpty,rec.size())` seems to prefer ` int[] `to `std::vector<int>`. At least it did not compile when I tried to use `std::vector<int>`

Reply:
```
comm.summ(&vec[0], vec.size());
```

Reply:
Indeed Dune only provides very limited support for containers. It is a rather thin wrappper around the MPI C bindings (hence the vectors). For vector simply use
```
comm.sum(vec.data, vec.size());
```
which will also work (without valgrind complaining) if the vector is empty.